webpack.config.js

```diff
@@ -0,0 +1,104 @@
+// By default eslint assumes packages imported are supposed to be dependencies,
```

Comment:
Good point. It really would not need to be a separate file. I will make that change.